Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM python:3.6-slim
+
+# Install Python tools (git + pipenv)
+RUN apt-get update && apt-get install -y git
+RUN pip install pipenv
+
+# Install memory_profiler if this script is run with PROFILE_MEMORY flag
+ARG INSTALL_MEMORY_PROFILER="false"
+RUN if [ "$INSTALL_MEMORY_PROFILER" = "true" ]; then \
+        apt-get update && apt-get install -y gcc && \
+        pip install memory_profiler; \
+    fi
+
+# Install plotly depedencies (orca and gcc)
+ARG ORCA_VERSION="1.2.1"
```

Comment:
Drive-by comment: I don't think we need this anymore, as we removed the graphs we were producing with plotly.

From the discussion today it sounded like revisiting graphs was low priority, and we might not use plotly/orca next time anyway, so maybe we can just remove

Reply:
I agree, have removed this.